ci_environment/docker/recipes/default.rb

```diff
@@ -18,10 +18,15 @@
   action :add
 end
 
-docker_pkg = 'docker-engine'
-# TODO docker_pkg += "-#{node['docker']['version']}" if node['docker']['version']
-
-package %W(linux-image-extra-#{`uname -r`.chomp} lxc #{docker_pkg})
+package %W(
+  linux-image-extra-#{`uname -r`.chomp}
```

Comment:
It is necessary for aufs support, but as we get into 4.0+ that has overlay in the kernel, aufs support is starting to go away (it's gone in Debian as of 4.0 but I'm not sure about Ubuntu).